src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2004,7 +2005,34 @@ bool entitiesQuery
   auto_ptr<DBClientCursor>  cursor;
   Query                     query(finalQuery.obj());
 
-  query.sort(BSON(ENT_CREATION_DATE << 1));
+  if (sorted == "")
```

Comment:
`sorted == ""` means that the `?sorted` parameter was not spedificed, i.e. CB should use default ordering which is based on creation date.

Or maybe I'm missing something in your point...